GiTinder/Services/UserServices.cs

```diff
@@ -1,9 +1,24 @@
 ﻿using GiTinder.Data;
+<<<<<<< HEAD
+using GiTinder.Models;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
 using System;
+using System.Collections.Generic;
 using System.Linq;
+using System.Net.Http;
+using System.Threading.Tasks;
 
 namespace GiTinder.Services
 {
+
+=======
+using System;
+using System.Linq;
+
+namespace GiTinder.Services
+{
+>>>>>>> ba9123ab47868405d039c3851fbcfee816e84dc4
```

Comment:
@Riceqrisp remove conflicts

Reply:
removed and changed namespace to GiTinder.Services.GitResponses